docs/data/sdks/java/index.md

```diff
@@ -46,7 +46,24 @@ Amplitude client = Amplitude.getInstance("YOUR_INSTANCE_NAME");
 client.init("YOUR_API_KEY");
 ```
 
-### Configure batching behavior
+### Configuration
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `serverUrl` | The server url events are uploaded to. | `https://api2.amplitude.com/2/httpapi` |
+    | `isBatchMode` |  Whether to use batch api. | `false` |
+    | `logMode` |  The level at which to filter out debug messages. | `LogMode.ERROR` |
+    | `eventUploadThreshold` |  SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach eventUploadPeriodSeconds interval. | `10` |
+    | `eventUploadPeriodMillis` | The amount of time SDK will attempt to upload the unsent events to the server or reach eventUploadThreshold threshold. | `10 seconds` |
+    | `callbacks` | Event callback which are triggered after event sent. | `null`|
+    | `proxy` | custom proxy for https requests. | `Proxy.NO_PROXY`|
+    | `flushTimeout` | events flushing thread timeout in milliseconds. | `0` |
+    | `options.minIdLength` | The minimum length for user id or device id. | `5` |
+    | `options.headers` | The custom headers. | `{"Content-Type", "application/json",
```

Comment:
The format show in preview is a bit broken, can you fix this? https://pr-634.d19s7xzcva2mw3.amplifyapp.com/data/sdks/java/#configuration